theseus_gui/src/style.css

```diff
@@ -1,15 +1,11 @@
-:root {
-  font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: 400;
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
we have our own inter.css on the modrinth CDN: https://cdn-raw.modrinth.com/fonts/inter/inter.css. though like nuxt 3 I think this should be bundled directly with the application (we don't need to request it every time)

Reply:
Added to the application.